example_run.m

```diff
@@ -0,0 +1,16 @@
+% Initialize a neuron session and call some hoc functions.
+
+% Initialization.
+clib.nrnmatlab.initialize();
+
+% Run HOC code.
+clib.nrnmatlab.hoc_run(3.14);
+
+% Run HOC code.
+% clib.nrnmatlab.fadvance(); % Crashes.
+
+% Done.
+clib.nrnmatlab.close();
+
+% Show results.
+disp(fileread('stdout.txt'));
```

Comment:
The stderr.txt is not yet displayed. Might even make it a fancy red, with fprintf(2, '%s\n', fileread('stderr.txt'))

Reply:
Resolved in latest commit.